sql2.txt

```diff
@@ -2,51 +2,84 @@
 to join the OrderDetails table with the product table.  For TotalCount, you will Use
 SUM(Quantity) AS TotalCount .  Order by product name.
 You will need to group by product ID.  Paste your SQL statement below.
+SELECT Products.ProductName, SUM(OrderDetails.Quantity) AS TotalCount
+FROM Products
+    JOIN OrderDetails ON Products.ProductID = OrderDetails.ProductID
+    GROUP BY Products.ProductID, Products.ProductName
```

Comment:
Since the ProductID is unique, there's no benefit in additionally grouping by ProductName. In fact, it generally makes the database work harder.

```suggestion
    GROUP BY Products.ProductID
```